OpenRA.Mods.Common/Traits/SupportPowers/NukePower.cs

```diff
@@ -99,6 +103,24 @@ class NukePowerInfo : SupportPowerInfo
 		[Desc("Amount of time after detonation to remove the camera.")]
 		public readonly int CameraRemoveDelay = 25;
 
+		[Desc("Corresponds to `Type` from `FlashPaletteEffect` on the world actor.")]
+		public readonly string FlashType = null;
+
+		[Desc("Range circle color.")]
+		public readonly Color CircleColor = Color.FromArgb(128, Color.Red);
+
+		[Desc("Range circle width.")]
+		public readonly float CircleWidth = 1;
+
+		[Desc("Range circle border color.")]
+		public readonly Color CircleBorderColor = Color.FromArgb(64, Color.Red);
+
+		[Desc("Range circle border width.")]
+		public readonly float CircleBorderWidth = 3;
+
+		[Desc("Render circles based these distance ranges while targeting.")]
+		public readonly WDist[] CircleRanges = null;
```

Comment:
Can/should we `FieldLoader.Required` this, since the trait is effectively nothing if there are no ranges defined?

Reply:
No, it should stay optional.